src/debugger/StopEventManager.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/* @flow */
+
+import invariant from "./../invariant.js";
+import type { DebugChannel } from "./channel/DebugChannel.js";
+import { Breakpoint } from "./Breakpoint.js";
+import { Stepper, StepIntoStepper, StepOverStepper } from "./Stepper.js";
+import { BabelNode } from "babel-types";
+
+export type StoppableObject = Breakpoint | Stepper;
+
+export class StopEventManager {
```

Comment:
Add comments for this class.

Reply:
These comments add no value. Try harder. :-)